influxdb3/tests/server/auth.rs

```diff
@@ -143,7 +145,7 @@ async fn auth() {
     assert_eq!(
         client
             .get("http://127.0.0.1:8181/api/v3/query_sql?db=foo&q=select+*+from+cpu")
-            .header("Authorizon", "Bearer foo")
+            .header("Authorizon", format!("Bearer {TOKEN}"))
```

Comment:
Hah, this one threw me for a minute. Total non-blocker, but maybe so `s/Authorizon/@uthorization/` so it is more immediately obvious the header is mispelled? Feel free to disregard :)

Reply:
I couldn't use @ because the code that parses headers checks for invalidity so I just shortened it to auth all lowercase to make it a bit more obvious